### ISSUE TITLE
Document [clrAlertIcon] option for specifying a custom icon for an alert

### ISSUE THREAD

**Issue by emmalivingstone** [vmware/clarity#2483]
```
[ ] bug
[x] feature request
[ ] enhancement
```

### Expected behavior

Angular alerts should be able to accept a `[clrAlertIcon]` option in case you want to use a custom icon for alerts, as you can in the static HTML alerts.

### Actual behavior

No option available, and If you add an `.alert-icon-wrapper` inside the Angular alert component you get two icons as demonstrated in the Stackblitz app below.

### Reproduction of behavior

https://clarity-light-theme-v11-x42wbq.stackblitz.io

### Environment details

* **Angular version:** 5.2.1

* **Clarity version:** 0.12.5

* **OS and version:** Windows 10

* **Browser:** [all]

**Comment by youdz** [comment]
They already do accept a `[clrAlertIcon]` input, but I'm surprised to see it's not documented. It was in the release notes at the time we added it, but apparently didn't make it to the actual documentation:

>Clarity alerts now have a [clrAlertIcon] input that accepts a string which represents the name of the icon in the Clarity Icons library. By default, Clarity will give your alert an icon based on the type of alert it is but this default can be easily overridden by assigning an icon name to [clrAlertIcon].
```html
<clr-alert clrAlertIcon="notification" clrAlertType="info" [clrAlertClosable]="false">
    <clr-alert-item>
        <span class="alert-text">
            This alert will have a bell instead of an "i" inside of a circle.
        </span>
    </clr-alert-item>
</clr-alert>
```

I'm renaming this issue as a documentation issue, thanks for raising this. 👍

**Comment by gnomeontherun** [comment]
Closed via #2507
